dispatch/dispatch.h

```diff
@@ -49,7 +49,7 @@
 
 #if defined(__linux__) && defined(__has_feature)
 #if __has_feature(modules)
-#include <stdio.h> // for off_t (to match Glibc.modulemap)
+//#include <stdio.h> // for off_t (to match Glibc.modulemap)
```

Comment:
Sure.  Like I said, this is a starting point for the conversation.  I know that the header is necessary for some cases, and I'd like understand that so we can come up with a solution.  This is the minimal change that fixes it for arm.  There are a ton of possible solutions of varying invasiveness.